bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcPostgresqlDAO.java

```diff
@@ -0,0 +1,210 @@
+	}
+	
+	/**************
+	 * ITEMS DAOs *
+	 **************/
+	@Override
+	public ItemsVO doCreateItemsTableIfNot(ItemsVO vo){
+		String sql = StringUtilsExt.replaceArrayMerge(
+				SQL_CREATE_ITEMS_TABLE_IF_NOT, 
+				new String[]{ "#itemsManageTable#", "#colname#", "#coltype#", "#itemsManageTable#"}, 
+				new String[]{ vo.getItemsManageTable(), vo.getColname(), vo.getColtype(), vo.getItemsManageTable()}
+				);
+		logger.debug("JDBC::doCreateItemsTableIfNot sql={}", sql);
+		Yank.execute(sql, null);
+		return vo;
+	}	
+	
+	@Override
+	public Long doCreateNewEntryInItemsTable(ItemsVO vo) {
+		String sql = StringUtilsExt.replaceArrayMerge(
+				SQL_CREATE_NEW_ENTRY_IN_ITEMS_TABLE, 
+				new String[]{ "#itemsManageTable#", "#itemname#"}, 
+				new String[]{ vo.getItemsManageTable(), vo.getItemname()}
+				);
+		logger.debug("JDBC::doCreateNewEntryInItemsTable sql={}", sql);
+		return Yank.insert(sql, null);
+	}
+	
+	@Override
+	public List<ItemsVO> doGetItemTables(ItemsVO vo) {
+		String sql = StringUtilsExt.replaceArrayMerge(
+				SQL_GET_ITEM_TABLES, 
+				new String[]{ "#itemsManageTable#"}, 
+				new String[]{ vo.getItemsManageTable()}
+				);
+		logger.debug("JDBC::doGetItemTables sql={}", sql);
+	    return Yank.queryBeanList(sql, ItemsVO.class, null);
+	}
+	 
+	/*************
+	 * ITEM DAOs *
+	 *************/
+	@Override
+	public void doStoreItemValue(Item item, ItemVO vo) {
+		vo = storeItemValueProvider(item, vo);
+		String sql = StringUtilsExt.replaceArrayMerge(
+				SQL_INSERT_ITEM_VALUE, 
+				new String[]{ "#tableName#", "#dbType#"}, 
+				new String[]{ vo.getTableName(), vo.getDbType()}
+				);
+		Object[] params = new Object[] { vo.getValue()};
+		logger.debug("JDBC::doStoreItemValue sql={} value='{}'", sql, vo.getValue() );
+		Yank.execute(sql, params);
+	}
+	
+	@Override
+	public List<HistoricItem> doGetHistItemFilterQuery(Item item, FilterCriteria filter, int numberDecimalcount, String table, String name) {
+		String sql = histItemFilterQueryProvider(filter, numberDecimalcount, table, name);
+		logger.debug("JDBC::doGetHistItemFilterQuery sql={}", sql);
+		List<Object[]> m = Yank.queryObjectArrays(sql, null);
+
+		List<HistoricItem> items = new ArrayList<HistoricItem>();
+		for(int i=0; i<m.size(); i++){
+			items.add(new JdbcItem(item.getName(), getState(item, m.get(i)[1]), objectAsDate(m.get(i)[0]) ));
+		}
+	    return items;
+	}
+	 
+	/****************************
+	 * SQL generation Providers *
+	 ****************************/
+	/**
+	 * @param filter
+	 * @param numberDecimalcount
+	 * @param table
+	 * @return
+	 */
+	private String histItemFilterQueryProvider(FilterCriteria filter, int numberDecimalcount, String table, String simpleName) {
+		logger.debug("JDBC::getHistItemFilterQueryProvider filter = {}, numberDecimalcount = {}, table = {}, simpleName = {}", filter.toString(), numberDecimalcount, table, simpleName);
+		
+		SimpleDateFormat jdbcDateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
This DateFormat object never changes. Maybe make it static final in the class scope and save a bit of performance?